shared/css/color.scss

```diff
@@ -54,12 +54,17 @@ $inset_color: $lighter_gray;
 $dark_color:  $purple;
 $hdr_color:   $purple;
 
-// some extra colors
+// Some extra colors.
 $red: #c00;
-$realyellow: yellow;
+$realgreen: #008000;
```

Comment:
We currently follow ['HexLength' style: short](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#hexlength) but we could switch to always 6 for consistency.  Note that `#008000` can't be shortened.

Do you prefer 6 hex digits for all colors?